src/pages/[platform]/build-ui/formbuilder/lifecycle/index.mdx

```diff
@@ -7,7 +7,7 @@ export const meta = {
                 'javascript',
                 'react'
               ]
-}
+};
```

Comment:
Added this because the new regex for generateDirectory.mjs is now looking for the semicolon

Reply:
Is this something that might trip up content authors (or just us if we forget)? I'm wondering since these files don't get prettier'ed and the lack of a semi-colon wouldn't cause a build failure or anything (I think?) to let someone know that it's missing; in that case you wouldn't know that the canonical URL is working for that page?

Reply:
Thanks for the feedback Heather! I think the generateDirectory.mjs will fail, but you're right it might trip up authors.  I think I can make sure that the error message asks them to look at how the meta object is formed


Yea at the moment we don't have a good way to know that the canonical URL is working for that page technically. Hoping we can figure something out in the future though, but for now I think we're trying to just update these based off of the list we were given for pages with duplicated content